src/core/qgsvectorlayerutils.h

```diff
@@ -312,6 +312,12 @@ class CORE_EXPORT QgsVectorLayerUtils
      * \since QGIS 3.12
      */
     static QHash<QString, QSet<QgsSymbolLayerId>> symbolLayerMasks( const QgsVectorLayer * ) SIP_SKIP;
+
+    /**
+     * \return the \a layer \a feature display string
```

Comment:
```suggestion
     * \returns a descriptive string for a \a feature, suitable for displaying to the user.
     *         The definition is taken from the ``displayExpression`` property of \a layer.
```

I think this deserves a tiny bit more information.